cli/tests/unit/blob_test.ts

```diff
@@ -69,6 +70,7 @@ unitTest(function nativeEndLine(): void {
 
   assertEquals(blob.size, Deno.build.os === "windows" ? 12 : 11);
 });
+*/
```

Comment:
I remembered you created an issue with all the tests to fix? Can you link me to it?

Reply:
https://github.com/denoland/deno/issues/7540